src/DssSpell.t.base.sol

```diff
@@ -2572,28 +2573,16 @@ contract DssSpellTestBase is DSTest, DSMath {
         }
     }
 
-    address[] deployerAddresses = [
-        0xdDb108893104dE4E1C6d0E47c42237dB4E617ACc,
-        0xDa0FaB05039809e63C5D068c897c3e602fA97457,
-        0xda0fab060e6cc7b1C0AA105d29Bd50D71f036711,
-        0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf,
-        0x0048d6225D1F3eA4385627eFDC5B4709Cab4A21c,
-        0xd200790f62c8da69973e61d4936cfE4f356ccD07,
-        0xdA0C0de01d90A5933692Edf03c7cE946C7c50445,
-        0x4D6fbF888c374D7964D56144dE0C0cFBd49750D3,  // Oracles
-        0x1f42e41A34B71606FcC60b4e624243b365D99745,  // Oracles
-        0x075da589886BA445d7c7e81c472059dE7AE65250   // Used for Optimism & Arbitrum bridge contracts
-    ];
-
     function checkWards(address _addr, string memory contractName) internal {
-        for (uint256 i = 0; i < deployerAddresses.length; i ++) {
+        AddressesDeployers deployers = new AddressesDeployers();
```

Comment:
we could move this to L71 along with `Rates` and `Addresses` and consider adding `Wallets` as well.

Reply:
I'm happy to do that.  My thinking was that since they are only used here it was better to make them local vars/inits rather than doing it in the setup.  But I do like the consistency of having them all in one place